freqtrade/strategy/interface.py

```diff
@@ -760,37 +804,72 @@ def advise_indicators(self, dataframe: DataFrame, metadata: dict) -> DataFrame:
         else:
             return self.populate_indicators(dataframe, metadata)
 
-    def advise_buy(self, dataframe: DataFrame, metadata: dict) -> DataFrame:
+    def advise_enter(
+        self,
+        dataframe: DataFrame,
+        metadata: dict,
+        is_short: bool = False
```

Comment:
This parameter is unnecessary in my eyes.

I think we reached an agreement in #2183 - where we concluded that we'll have 2 methods instead of 4.

Obviously, users will need 4 columns  (or 6, if we include the "tag" columns) - but the seperation to 4 columns will only confuse users without real benefit.

Reply:
no it doesn't ... this is just telling you which type of signals you're populating (which method is called) - not  if you're entering a long, short or nothing.